script.js

```diff
@@ -47,4 +50,119 @@ buttonElement.addEventListener('click', function (event) {
     subElement.classList.toggle('active');
 });
 
+buttonRight.addEventListener('click', function (){
+    move('right');
+}); 
+buttonLeft.addEventListener('click', function (){
+    move('left');
```

Comment:
Way too generic name, what does move do? Please make function names more descriptive in order to make code more maintainable

Reply:
Changed name to moveCarousel.

Reply:
Is it the carousel that moves? Or the slide?

Reply:
Can change to moveCarouselSlides

Reply:
I do not see the changes function is still called move()

Reply:
done